package.json

```diff
@@ -68,7 +68,7 @@
     "sass-loader": "11.1.1",
     "standard": "16.0.4",
     "style-loader": "2.0.0",
-    "webpack": "5.74.0",
+    "webpack": "5.76.2",
```

Comment:
Is this just an update to the latest version?

Reply:
Was a vulnerability to had to bump the version.  Tested it and works just the same.

Reply:
This the same reason for the bump to the sequelize version?